### PR TITLE
 OPENEUROPA-1675: Provide fallback title for inpage navigation variant.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: tests/Kernel/Paragraphs/ContentRowTest.php [hunk 1]
```diff
@@ -190,6 +190,23 @@ public function testSideMenuTranslation(): void {
     $left_column = $crawler->filter('.ecl-row .ecl-col-md-3.ecl-u-z-navigation');
     $this->assertContains('French page navigation', $left_column->html());
     $this->assertContains('French rich text title', $left_column->html());
+
+    // Create the main content row paragraph with empty title.
```
Comment (imanoleguskiza) [inline tests/Kernel/Paragraphs/ContentRowTest.php]: Its a bit weird to add this test inside the paragraph translation test, but I would like to have the next reviewers input before asking for changes.

Comment (nagyad) [reply tests/Kernel/Paragraphs/ContentRowTest.php]: We are testing if the fallback title gets translated as well.

**Comment by nagyad** [comment]
Rebased.

File: tests/Kernel/Paragraphs/ContentRowTest.php (hunk 1 above)
Comment (imanoleguskiza) [reply tests/Kernel/Paragraphs/ContentRowTest.php]: Fair enough :+1:

Comment (sergepavle) [reply tests/Kernel/Paragraphs/ContentRowTest.php]: Yes. I agree. It is make sense to move this code in other place.

**Comment by nagyad** [comment]
Did rebase.

**Pull request merged**